meson.build

```diff
@@ -0,0 +1,49 @@
+project('hack-components', version: '0.0.0')
+
+conf = configuration_data()
+
+cmd = run_command('sh', '-c', 'echo $FLATPAK_ARCH')
+conf.set('FLATPAK_ARCH', cmd.stdout().strip())
+
+cmd = run_command('sh', '-c', 'echo $BRANCH')
+conf.set('BRANCH', cmd.stdout().strip())
```

Comment:
I still would like to get these environment variables out of the way if possible... At least for `$BRANCH` we can pass it as an actual meson configuration option, no?

Reply:
Yes, we could.